sys/linux/auto.txt

```diff
@@ -2001,39 +2004,39 @@ dccp_timeout_nla_policy$auto_nf_conntrack_proto_dccp [
 ] [varlen]
 
 devconf_ipv4_policy$auto_devinet [
-	NETCONFA_IFINDEX			nlattr[NETCONFA_IFINDEX, intptr]
-	NETCONFA_FORWARDING			nlattr[NETCONFA_FORWARDING, intptr]
-	NETCONFA_RP_FILTER			nlattr[NETCONFA_RP_FILTER, intptr]
-	NETCONFA_PROXY_NEIGH			nlattr[NETCONFA_PROXY_NEIGH, intptr]
-	NETCONFA_IGNORE_ROUTES_WITH_LINKDOWN	nlattr[NETCONFA_IGNORE_ROUTES_WITH_LINKDOWN, intptr]
+	NETCONFA_IFINDEX			nlattr[NETCONFA_IFINDEX, array[int8, 4]]
```

Comment:
Looks very weird.. Maybe keep `int32` for all 4 byte `NLA_UNSPEC` types and `int64` for all 8 bytes?

Reply:
Ideally it should be `ifindex` type.